createFolders.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+PROJECT_NAME=data-and-algorithms
+DIRECTORY=${PWD}
+BASE_DIRECTORY=$(echo "$DIRECTORY" | rev | cut -d "/" -f1 | rev)
+
+if [[("$1" != "")]]; then
+   if [[("$BASE_DIRECTORY" != "$PROJECT_NAME")]]; then
+      echo "Make sure you are in $PROJECT_NAME project"
+      exit
+   fi
+   rootfileName=$1
+   mkdir ./$rootfileName
+   cp -a ./Python/ ./$rootfileName/
+   rm ./$rootfileName/*/*/*.py
```

Comment:
maybe it's better to not copy the files in the first place:

```shell
rsync --archive --include='*/' --exclude='*' "${DIRECTORY}/Python/" "${DIRECTORY}/${programmingLanguage}"
```

source: https://stackoverflow.com/a/40586240/688337

or if you want to avoid rsync for whatever reason

```shell
(cd "${DIRECTORY}/Python/" && \
 find . -type d -depth -exec mkdir -p "${DIRECTORY}/${programmingLanguage}/{}" \;)